### PR TITLE
This fixes HYRAX-1426: the spurious PPTConnection error messages.

### PR DESCRIPTION
**Pull request by jgallagher59701**
These messages happen because the olfs no longer sends an 'exit' message to the BES. Instead the OLFS just closes the socket. The PPTConnection code detects this as a read of zero bytes.

### CODE REVIEW

**Comment by jgallagher59701** [comment]
Note: In the logs this will look like:

```
2024-05-30T19:18:56 UTC|&|50291|&|request|&|get.dmr|&|dap2|&|/Users/jimg/src/opendap/hyrax/build/sha
re/hyrax/data/nc/fnoc1.nc
2024-05-30T19:18:56 UTC|&|50291|&|info|&|PPTConnection::receive: read EOF from the OLFS, beslistener
 exiting.2024-05-30T19:18:56 UTC|&|50291|&|info|&|Received exit command.
2024-05-30T19:18:56 UTC|&|50130|&|info|&|beslistener child pid: 50291 exited with status: 6; num chi
ldren: 0
2024-05-30T19:18:56 UTC|&|51338|&|request|&|get.dmr|&|dap2|&|/Users/jimg/src/opendap/hyrax/build/sha
re/hyrax/data/nc/fnoc1.nc
```
Code 6 id the normal exit status for one of the child listener processes.

File: ppt/PPTConnection.cc [hunk 1]
```diff
@@ -273,8 +267,20 @@ bool PPTConnection::receive(map<string, string> &extensions, ostream *strm)
 	// if x then extensions follow, if d then data follows.
 	int bytesRead = readChunkHeader(_inBuff, 8);
 	BESDEBUG( MODULE, prolog << "Reading header, read " << bytesRead << " bytes" << endl );
-	if (bytesRead != 8)
-		throw BESInternalError(prolog + "Failed to read chunk header", __FILE__, __LINE__);
+    if (bytesRead == 0) {
+        INFO_LOG("PPTConnection::receive: read EOF from the OLFS, beslistener exiting.");
```
Comment (ndp-opendap) [inline ppt/PPTConnection.cc:271]: This log line needs to be terminated with a _`\n`_

Comment (jgallagher59701) [reply ppt/PPTConnection.cc:271]: Thanks.

**Review by ndp-opendap — CHANGES_REQUESTED** [review]
Looks good but the log line `PPTConnection::receive: read EOF from the OLFS, beslistener
 exiting.` needs a newline thingy.

**Review by ndp-opendap — APPROVED** [review]
Pass the tests and :shipit:

**Pull request merged**